### DISCUSSION TITLE
2021/10/30/managing-python-on-macos-monterey/

### DISCUSSION THREAD

**Discussion by giscus** [russmckendrick/blog#9] [Comment]
# 2021/10/30/managing-python-on-macos-monterey/

This week was not only the release of macOS Monterey, but it was also new Mac Book Pro week for a lot of people, including myself. Given that it was alot of effot to install Python on macOS Big Sur I was a little worried that it would take the same amount of effort.
As I was starting a with a clean macOS Monterey installation and had already installed Homebew I just needed to run the following command to install pyenv:

https://www.mediaglasses.blog/2021/10/30/managing-python-on-macos-monterey/

**Comment by S3RG404** [comment]
Everything went alright until ansible --version

 ~ % ansible --version
zsh: command not found: ansible

**Reply by kbricen** [reply]
Did you actually opened the new shell, before running the command? it worked for me.

**Reply by russmckendrick** [reply]
What do you get when running ...

```
python --version
which python
```

... it sounds like the wrong version of Python maybe active?

**Reply by S3RG404** [reply]
****@MBP-**** ~ % python --version
Python 3.10.1
*****@MBP-***** ~ % which python
/Users/*****/.pyenv/shims/python

**Comment by xinfengmich** [comment]
% pip install --user ansible
Collecting ansible
....
Using legacy 'setup.py install' for ansible, since package 'wheel' is not installed.
Using legacy 'setup.py install' for ansible-core, since package 'wheel' is not installed.
Installing collected packages: resolvelib, PyYAML, pyparsing, pycparser, MarkupSafe, packaging, jinja2, cffi, cryptography, ansible-core, ansible
  Running setup.py install for ansible-core ... done
  Running setup.py install for ansible ... done
Successfully installed MarkupSafe-2.0.1 PyYAML-6.0 ansible-5.3.0 ansible-core-2.12.2 cffi-1.15.0 cryptography-36.0.1 jinja2-3.0.3 packaging-21.3 pycparser-2.21 pyparsing-3.0.7 resolvelib-0.5.4

% ansible --version
zsh: command not found: ansible

% python --version          
Python 3.10.0

% which python              
/Users/xinfeng/.pyenv/shims/python

Tried in a new zsh shell.

**Reply by S3RG404** [reply]
same as me above. installed using pyenv.

**Reply by jmsy00** [reply]
you may just need to add $HOME/.local/bin to your path
